source/agora/utils/TxBuilder.d

```diff
@@ -71,18 +71,22 @@ module agora.utils.TxBuilder;
 import agora.common.Amount;
 import agora.common.Types;
 version (unittest) import agora.consensus.data.genesis.Test;
+import agora.consensus.data.Params: ConsensusConfig;
 import agora.consensus.data.Transaction;
 import agora.crypto.ECC;
 import agora.crypto.Hash;
 import agora.crypto.Key;
 import agora.script.Lock;
+import agora.script.Opcodes;
 import agora.script.Signature;
 /* version (unittest) */ import agora.utils.Test;
 
 import std.algorithm;
 import std.format;
 import std.range;
 
+private const ConsensusConfig consensus_config = ConsensusConfig.init;
```

Comment:
`private immutable Amount defaultMinFee = ConsensusConfig.init.minFee`.

Better not to have a full `ConsensusConfig` here, otherwise its usage will grow.

Reply:
Actually, I would remove this global and just use this as the default value for `TxBuilder.min_fee`

Reply:
But I want to use it in the unit tests also.